models/sipnet/R/write.restart.SIPNET.R

```diff
@@ -1,8 +1,37 @@
-write.restart.SIPNET<- function(nens,outdir,run.id,time,settings,prior,analysis){
-  for(i in 1:nens){   
-    file.rename(file.path(outdir,run.id[[i]],"sipnet.out"),file.path(outdir,run.id[[i]],paste0("sipnet.out",time[t])))
-    file.remove(file.path(settings$rundir,run.id[[i]],"sipnet.clim"))
-    do.call(my.write.config,args=list(defaults,list(pft=prior[i,],env=NA),
-                                      settings, run.id[[i]],inputs = settings$run,IC=analysis[i,-1]))   
+write.restart.SIPNET<- function(out.dir, runid, time, settings, analysis.vec,
+                                RENAME = TRUE, PLOT = FALSE, variables,
+                                sample_parameters = FALSE,
+                                trait.values = NA, my.write.config, met){
+ 
+  if(RENAME == TRUE) {
+    file.rename(file.path(outdir,runid,"sipnet.out"),
+                file.path(outdir,runid,paste0("sipnet.out",time)))
+    system(paste("rm",file.path(rundir,runid,"sipnet.clim")))
+  }else{
+    print(paste("Files not renamed -- Need to rerun year",time,"before next time step"))
   }
+   #file.remove(file.path(settings$rundir,runid,"sipnet.clim"))
+  
+  settings$run$start.date <- paste0(time + 1,"/01/01")
```

Comment:
this seems really fragile.  As we generalize we'll need to be passing the new end time as the argument (and assuming that the previous end time is the new start time.